app/services/rag_wise/chunking_service.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-class Rag::ChunkingService
+class RagWise::ChunkingService
```

Comment:
Squash services renaming in e24eb46521e6bb6fb2ddb5777ff19dd6a934c2c0 with original service definitions in 02b6809c9d55a545cb7cffb7682a5ab584c04302

Reply:
Squashed